person.rb

```diff
@@ -1,4 +1,11 @@
-class Person
+# rubocop:disable all
+class Nameable
+  def correct_name
+    raise NotImplementedError
+  end
+end
```

Comment:
- **[OPTIONAL]** Kindly move this class to its own file

Reply:
@DammyShittu I did that from the start but it doesn't work even tho I used the require_relative to import the nameable file so I'm having trouple there.